builder/virtualbox/ovf/config.go

```diff
@@ -87,15 +90,24 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 	errs = packer.MultiErrorAppend(errs, c.VBoxManagePostConfig.Prepare(&c.ctx)...)
 	errs = packer.MultiErrorAppend(errs, c.VBoxVersionConfig.Prepare(&c.ctx)...)
 
-	if c.SourcePath == "" {
+	// Get downloadable URL
+	downloadableURL, err := common.DownloadableURL(c.SourcePath)
+	notURL := !strings.HasPrefix(downloadableURL, "http://") && !strings.HasPrefix(downloadableURL, "https://")
+
+	// SourcePath can't be empty and has to be 'downlodable'
+	if err != nil {
+		errs = packer.MultiErrorAppend(
+			errs, fmt.Errorf("source_path is invalid: %s", err))
+	} else if c.SourcePath == "" {
```

Comment:
What happens if an empty path is passed to `downloadableURL`? Wondering if this state is ever valid